src/ichmod.cpp

```diff
@@ -144,45 +144,57 @@ main( int argc, char **argv ) {
 void
 usage() {
     char *msgs[] = {
-        "Usage: ichmod [-rhvVM] null|read|write|own userOrGroup dataObj|Collection ...",
+        "Usage: ichmod [-rhvVM] AccessLevel userOrGroup dataObject|Collection ...",
         " or    ichmod [-rhvVM] inherit Collection ...",
         " or    ichmod [-rhvVM] noinherit Collection ...",
-        " -r  recursive - set the access level for all dataObjects",
-        "             in the entered collection and subCollections under it",
+        " ",
+        "Options:",
+        " -r  recursive - set the AccessLevel for all dataObjects",
+        "             in the entered Collection and subCollections under it",
         " -v  verbose",
         " -V  Very verbose",
         " -M  Admin Mode",
         " -h  this help",
         " ",
-        "Modify access to dataObjects (iRODS files) and collections (directories).",
+        "Modify access to dataObjects and Collections.",
+        " ",
+        "By default, the original creator of a dataObject has 'own' permission.",
+        " ",
+        "The iRODS Permission Model allows for multiple ownership.",
+        "  ",
+        "The iRODS Permission Model is linear with 10 levels.",
+        "A granted AccessLevel also includes access to all lower AccessLevels.",
+        " ",
+        " - 'own'",
+        " - 'delete_object'",
+        " - 'write', 'modify_object'",
+        " - 'create_object'",
+        " - 'delete_metadata'",
+        " - 'modify_metadata'",
+        " - 'create_metadata'",
+        " - 'read', 'read_object'",
+        " - 'read_metadata'",
+        " - 'null'",
```

Comment:
These will likely need to be reverted to the versions that do not use the underscores because of backwards compatibility.

Reply:
Discush.   Want this to be the good version....

Reply:
discussed - [it should behave](https://github.com/irods/irods/pull/6397#pullrequestreview-974230895)